cloudera-integration/csd/scripts/control.sh

```diff
@@ -148,6 +155,20 @@ case $CMD in
     --zk-hosts $SOLR_ZK_QUORUM
     ;;
 
+  (start_log_collector_kafka)
+    echo "Starting Log Collector Kafka Mode on topic $KAFKA_TOPIC"
```

Comment:
We need to add some arguments to the CSD file (service.sdl) to expose these variables in cloudera manager